_includes/tutorials/console-consumer-read-specific-offsets-partition/confluent/markup/dev/consume-topic-partition-zero.adoc

```diff
@@ -0,0 +1,21 @@
+Next let's open up a console consumer to read records sent to the topic in the previous step, but you'll only read from the first partition. Kafka partitions
+are zero based so your two partitions are numbered `0`, and `1` respectively.
+
+Confluent Cloud CLI currently does not have the ability to specify reading from a particular partition, so for the next few steps you'll be using the console consumer built into a Docker image.
+Notice that you are passing in the path to the `configuration/ccloud.properties` file that you created earlier.
```

Comment:
Maybe not a big deal but at this point, the user hasn't created a `configuration` directory.

Reply:
This is actually true for all CCloud KTs.

Reply:
Not the ones I've done :) checkout out [cogrouping](https://kafka-tutorials.confluent.io/cogrouping-streams/confluent.html#initialize-the-project ) or [filtering](https://kafka-tutorials.confluent.io/filter-a-stream-of-events/confluent.html#initialize-the-project) as examples. You don't necessarily have to do add that step.

Reply:
🙇 I will fix ;)

Reply:
Done in latest commit